handbook/engineering/distribution/goals.md

```diff
@@ -2,7 +2,7 @@
 
 Goals are continuously updated and reviewed. If you find these goals do not reflect our current priorities or are out of date, please update them as soon as possible or add it as a topic to our [weekly sync](recurring_processes.md#weekly-distribution-team-sync).
 
-## Medium-term goals
+## Medium-term goals (~3-6 Months)
```

Comment:
Now that we have clear milestones for each goal (great!), can we delete this heading entirely and give a rough estimate per milestone in each goal? It would also be cool to mark down when a milestone was completed, which release cycle it was completed in.

Reply:
As discussed, I would like to keep this header and I have added length and milestone estimates.

Reply:
I love all the details you provided for each goal, and after reading through the rest of the PR I still don't think this heading is necessary, but you can keep it if you want.

Reply:
I think it will make more sense once we have other long-term goals, if it does not at that point, ill remove it.